.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI Checks
+on:
+  push:
+    branches: ["**"]
+  pull_request:
+    branches: [master]
```

Comment:
Wrong branch for this repo.

Reply:
Change it to main branch for pull_request